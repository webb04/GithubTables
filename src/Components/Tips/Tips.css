.Tips {
    text-align: center;
}

.tip {
    padding-top: 25px;
}