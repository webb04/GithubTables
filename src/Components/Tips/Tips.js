import React from 'react';

import './Tips.css';

export const Tips = () =>
  <div className="Tips">
    <div className="tip"><em>Tip: Hold shift when sorting to multi-sort!</em></div>
    <div className="tip"><em>Tip: You can drag column headers to resize!</em></div>
    <div className="tip"><em>Tip: Filters at the top!</em></div>
  </div>;
