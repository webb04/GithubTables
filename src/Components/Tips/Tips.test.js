import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { Tips } from './Tips';
import { shallow, mount } from 'enzyme';

it('Renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Tips />, div);
    ReactDOM.unmountComponentAtNode(div);
  });
  
it('Renders correctly', () => {
    const tree = renderer
        .create(<Tips />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});