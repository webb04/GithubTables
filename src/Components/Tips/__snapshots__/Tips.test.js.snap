// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="Tips"
>
  <div
    className="tip"
  >
    <em>
      Tip: Hold shift when sorting to multi-sort!
    </em>
  </div>
  <div
    className="tip"
  >
    <em>
      Tip: You can drag column headers to resize!
    </em>
  </div>
  <div
    className="tip"
  >
    <em>
      Tip: Filters at the top!
    </em>
  </div>
</div>
`;
